### DISCUSSION TITLE
file rename and update references

### DISCUSSION THREAD

**Discussion by chk1xn** [microsoft/vscode-discussions#876] [Extension Development QnA] [answered]
Hi community,

as I'm completely new to both, VSC and this community please bear with me and if this is not the right place to ask my Q, please let me know where I could post it.

After more than a decade I've moved from Dreamweaver to Visual Studio Code. The transition went very smooth and after just 2-3 hours I had all my site projects up and running using SFTP by Natizyskun.

So now I'm in midst of getting used to VSC, reading tons of documentation and watching many tutorials. It's fun.
What I'm mostly interested for now is something supposedly simple but I couldn't find an answer yet: renaming files and update all their references.

For example, an HTML file is referencing to css, js, images, audio, video, pdf and other files. When I rename such a file, Dreamweaver notified me that this file is referenced in other files and offered me to update all these references by the click of a button.
How can I accomplish this in VSC?
I have a feeling I'm missing something obvious here as this is such a common use case but simply right-click a file and rename it only changes its filename. period.

Thanks for any comments.

**Comment by tamuratak** [comment, marked as answer]
This forum is for developers of VS Code extensions.

Your feature request for the built-in extension would be appropriate for https://github.com/Microsoft/vscode. It might be a duplicate of microsoft/vscode/issues/181935.

**Reply by chk1xn** [reply]
Thank you. Will continue there...
